tests/Integration/Behaviour/Features/Scenario/Product/duplicate_product.feature

```diff
@@ -216,8 +262,25 @@ Feature: Duplicate product from Back Office (BO).
     And product copy_of_product1 should have 1 customizable text field
     And product copy_of_product1 should have 0 customizable file fields
     And product "copy_of_product1" should have 1 specific prices
-#@todo: assert stock info
-#@todo: add tests for other type of products Pack, Virtual, Combinations
+    And product "copy_of_product1" should have following stock information:
+      | minimal_quantity    | 12         |
+      | quantity            | 0          |
+      | location            |            |
+      | low_stock_threshold | 42         |
+      | low_stock_alert     | true       |
+      | available_date      | 1969-07-16 |
+    And product "copy_of_product1" localized "available_now_labels" should be:
+      | locale | value      |
+      | en-US  | get it now |
+      | fr-FR  |            |
+    And product "copy_of_product1" localized "available_later_labels" should be:
+      | locale | value        |
+      | en-US  | too late bro |
+      | fr-FR  |              |
+    And product "copy_of_product1" should have following images:
```

Comment:
ping @jolelievre this scenario is failing after your changes in ProductImageFeatureContext::assertProductImages.
Your changes makes perfect sense for all other checks except duplication. In duplication scenarios we don't know the ids of images.
One solution is to add additional method dedicated for duplicated product images check (that would do the same as it previously did in ProductImageFeatureContext::assertProductImages).
Do you have any alternatives?